test/test_data/dhclient.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+mapping_test_data = [
+    {
+        "product": "dhcp",
+        "version": "4.4.3",
+        "version_strings": [r'"name":"dhcp","version":"4.4.3"'],
+    },
+]
+package_test_data = [
+    {
+        "url": "https://www.rpmfind.net/linux/fedora/linux/development/rawhide/Everything/aarch64/os/Packages/d/",
+        "package_name": "dhcp-client-4.4.3-5.P1.fc38.aarch64.rpm",
+        "product": "dhcp",
+        "version": "4.4.3",
+    },
+]
+
+# This checker only supports .rpm distros as no useful version patterns were found for .deb
```

Comment:
I like the comment here, but let's also have one in the checker itself.  Reasoning: I think eventually we'll have this as checker metadata so if you know you're scanning a set of .deb packages you can disable checkers that don't work there.  Plus I think people are more likely to check the checker file if they're not sure why something's not working on their system.